src/main/resources/messages.properties

```diff
@@ -9,6 +10,8 @@ field.updated={0} was successfully updated
 password.invalid.format=Provided password is not strong enough
 project.field.already.exists=Project with {0} {1} already exists
 project.created=Project has been created
+project.invitation.acceptance=You have been successfully added to {0}!
+project.no.role=Inviting is forbidden for you
 registration.successful=Registration was successful. Activation link was sent to email address.
```

Comment:
Project no rolę brzmi bardzo ogólnie a dotyczy tylko zaproszenia. Dałbym np. you have no permission for requested operation